src/Models/Site.php

```diff
@@ -332,12 +332,15 @@ public function serialize()
     {
         $settings = $this->get('settings');
 
-        $created_date = is_numeric($created = $this->get('created')) ? $created : strtotime($created);
+        $date_format = $this->getConfig()->get('date_format');
+        $format_date = function ($attribute) use ($date_format) {
+            return date($date_format, is_numeric($date = $this->get($attribute)) ? $date : strtotime($date));
```

Comment:
There is a lot happening in this one line. I would break this up into steps. That'll make it easier to comprehend and debug at the cost of a few linebreaks. 3/10 though.